packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4275,6 +4276,8 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return this.reconnectionManager
       .reconnect(options)
+      .then(() => this.waitForRemoteSDPAnswer())
```

Comment:
nitpick: waitForRemoteSDPAnswer() relies on `deferSDPAnswer` being created when we get LOCAL_SDP_OFFER_GENERATED event. In theory it is possible that this event is not sent and then we would be using the old value of `deferSDPAnswer` that was resolved during initial join, so waitForRemoteSDPAnswer() would resolve even though no offer or answer is sent/received. I guess in such theoretical case the call to `meeting.mediaProperties.webrtcMediaConnection.reconnect()` in reconnectMedia() should have also failed, so maybe we can just ignore this.